modules/op-changing-default-service-account.adoc

```diff
@@ -0,0 +1,26 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: CONCEPT
+[id="op-changing-default-service-account_{context}"]
+= Changing the default service account
+
+[role="_abstract"]
+Change the default service account by configuring the `default-service-account` option in `.spec.pipeline` and `.spec.trigger` parameters. By default, the service account name is `pipeline`.
```

Comment:
@shipsing By the example I wanted to present how the user can change it to `pipelines` instead of `pipeline`. But I can see now how that can be confusing. The default is `pipeline` though, so I will change the example in YAML.